sdk/template/azure-template/setup.py

```diff
@@ -42,7 +42,7 @@
     license='MIT License',
     # ensure that the development status reflects the status of your package
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        "Development Status :: 5 - Production/Stable",
```

Comment:
I dont think we want to make this production stable. Can you please revert this change?